### DISCUSSION TITLE
Updates to the tutorial

### DISCUSSION THREAD

**Discussion by slambert** [lexiconhq/lexicon#73] [Troubleshooting and Q&A]
I'm going through the tutorial and running into some issues that may need updating. I'm seeing lots of warnings of deprecated packages on these installs but more specifically...

[Install the expo cli](https://docs.lexicon.is/version-2.2.0/tutorial/setup#install-the-expo-cli)

Running `expo --version` after this gives a warning:

```
> expo --version
WARNING: The legacy expo-cli does not support Node +17. Migrate to the new local Expo CLI: https://blog.expo.dev/the-new-expo-cli-f4250d8e3421.
6.3.10
```

I'm running Node 18

```
> node --version
v18.20.4
```

I visited the [expo-dev site listed in the warning](https://blog.expo.dev/the-new-expo-cli-f4250d8e3421) and saw

> Usage
> 
> The most important difference is that users must now invoke npx expo start instead of expo start to develop an app — users also no longer need to run npm install -g expo-cli as the CLI is shipped with the expo package.

So I removed expo-cli with `npm remove --global expo-cli`

Then ran

```
> npx expo --version
0.18.29
```

This seemed to allow me to move forward...

**Comment by ismailzabalawi** [comment]
I am struggling with this too.
